wurst/systems/units/UnitEntity.wurst

```diff
@@ -11,7 +11,7 @@ public function randomAngle() returns angle
 
 
 public abstract class UnitEntity
-    static thistype array instances
+    static HashMap<unit, thistype> instances = new HashMap<unit,thistype>()
```

Comment:
What is the overhead on this?

Reply:
No idea to be honest but I'd like to try it to see if its more reliable than using unitindex